### PR TITLE
[CORE-9689] Add support for absolute url redirect

### PR DESCRIPTION
**Pull request by iamuchejude**
https://jira.nebenan.de/browse/CORE-9689

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "homepage": "https://goodhood.eu",
   "repository": "goodhood-eu/react-router-manager",
   "bugs": "https://github.com/goodhood-eu/react-router-manager/issues",
-  "version": "3.2.2",
+  "version": "3.2.3-beta.0",
```
Comment (inomn) [inline package.json:9]: looks more like a new feature, go with 3.3.0?

**Review by eliasbraun — CHANGES_REQUESTED** [review]

File: src/render.jsx [hunk 2]
```diff
@@ -1,5 +1,10 @@
 import { Switch, Route, Redirect } from 'react-router';
-import { isNumber, isFunction, getMatchableRoute } from './utils';
+import { isNumber, isFunction, getMatchableRoute, isAbsoluteUrl } from './utils';
+
+export const AbsoluteRedirect = ({ to }) => {
+  window.location.replace(to);
```
Comment (inomn) [inline src/render.jsx:5]: what about SSR?

File: src/utils.jsx [hunk 3]
```diff
@@ -5,3 +5,5 @@ export const getMatchableRoute = (route) => {
   const { exact = true, sensitive = false, strict = true, ...params } = route;
   return { ...params, exact, sensitive, strict };
 };
+
+export const isAbsoluteUrl = (url) => url && url.startsWith('http');
```
[the remark this answers is not in the document]
Comment (peterhass) [reply src/utils.jsx:9]: No, absolute url needs to match url spec schema 100%

schema://host/path?query etc

But another thing: Better to check for http:// and https:// to not get into weird cases with weird route names

**Comment by inomn** [comment]
We don't need it anymore

**Pull request closed without merging**